scripts/dts/gen_defines.py

```diff
@@ -623,8 +623,6 @@ def write_vanilla_props(node):
                 if isinstance(subval, str):
                     macro2val[macro + f"_IDX_{i}"] = quote_str(subval)
                     subval_as_token = edtlib.str_as_token(subval)
-                    macro2val[macro + f"_IDX_{i}_TOKEN"] = subval_as_token
-                    macro2val[macro + f"_IDX_{i}_UPPER_TOKEN"] = subval_as_token.upper()
```

Comment:
Needs macros.bnf update

Reply:
Can we do that as a follow PR, since none of the string TOKENs are documented in `macros.bnf`

Reply:
> Can we do that as a follow PR

OK